bliss/conf/base_config.yaml

```diff
@@ -109,8 +109,6 @@ encoder:
             flux_error:
                 _target_: bliss.encoder.metrics.FluxError
                 survey_bands: ${encoder.survey_bands}
-            galaxy_shape_error:
-                _target_: bliss.encoder.metrics.GalaxyShapeError
```

Comment:
Can you override this in your case-study-specific config file, rather than changing the base config file?

Reply:
Resolved